core/server/src/main/java/alluxio/master/file/meta/InodeFile.java

```diff
@@ -326,14 +327,15 @@ public synchronized void complete(long length)
   @Override
   public synchronized String toString() {
     StringBuilder sb = new StringBuilder("InodeFile(");
-    sb.append(super.toString()).append(", LENGTH: ").append(mLength);
-    sb.append(", Cacheable: ").append(mCacheable);
-    sb.append(", Completed: ").append(mCompleted);
-    sb.append(", Cacheable: ").append(mCacheable);
-    sb.append(", mBlocks: ").append(mBlocks);
-    sb.append(", mTtl: ").append(mTtl);
-    sb.append(")");
-    return sb.toString();
+    return sb.append(super.toString())
+        .append(", BLOCKS: ").append(mBlocks)
```

Comment:
This was for consistency with the super.toString() output, but I agree with you. I will make the change to toStringHelper.